traitsui/editors/boolean_editor.py

```diff
@@ -18,7 +18,7 @@
 
 from ..view import View
 
-from .text_editor import ToolkitEditorFactory as EditorFactory
+from .text_editor import TextEditor as EditorFactory
```

Comment:
why import this as `EditorFactory`?

Reply:
no good reason, really. cleaning these up as well

Reply:
cleaned up.